rpm/generic/zfs.spec.in

```diff
@@ -172,8 +172,12 @@ Obsoletes:      libzpool4
 This package contains the zpool library, which provides support
 for managing zpools
 
+%if 0%{?rhel} == 7 || 0%{?centos} == 7
 %post -n libzpool5 -p /sbin/ldconfig
 %postun -n libzpool5 -p /sbin/ldconfig
+%else
+ldconfig_scriptlets -n libzpool5
```

Comment:
Isn't ldconfig_scriptlets a macro (and so requires % prefix)?

Reply:
Oops, yes it is.  Updated.